CONTRIBUTING.md

```diff
@@ -0,0 +1,77 @@
+# How to Contribute
+
+CoreOS projects are [Apache 2.0 licensed](LICENSE) and accept contributions via
```

Comment:
s/CoreOS/Red Hat

Reply:
If you change it to "Red Hat projects are" it becomes a bit wrong, because not all our projects use  the Apache license.

I would suggest changing the words "CoreOS projects" to "The cluster-monitoring-operator".